app.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__, template_folder='templates')
+
+submissions = []
+
+
+@app.route('/')
+def index():
+    return render_template('input_website.html')
+
+
+@app.route('/submit', methods=["POST"])
+def submit():
+    if request.method == 'POST':
+        first_name = request.form.get('firstName').capitalize()
+        last_name = request.form.get('lastName').capitalize()
+        email = request.form.get('email')
+        url = request.form.get('url')
+        discount = request.form.get('discount')
+
+        data = {
+            'first_name': first_name,
+            'last_name': last_name,
+            'email': email,
+            'url': url,
+            'discount': discount
+        }
+
+        submissions.append(data)
+
+        print("Form Submitted:")
```

Comment:
does this print to the browser or the console?

Reply:
Prints to the console for the time being